notebooks/basics.ipynb

```diff
@@ -82,6 +82,9 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
    "version": "3.7.3"
+  },
+  "voila": {
+   "template": "default"
```

Comment:
We might want to think about a schema or a better structure before using the notebook metadata for voila.

Reply:
out of scope for now